docker-compose.yml

```diff
@@ -12,12 +12,12 @@ services:
       - "27017:27017"
 
   redis:
-    image: redis
+    image: redis:5.0.7
```

Comment:
We used `latest` for a while, at least in this docker compose example), so shuld be safe to tie to version `6`.